### PR TITLE
Add blog post for 0.2.0 release announcement

### LINKED ISSUE #80: Blog Post: Announcing 0.2
We should have a blog post on kubeflow.org announcing the 0.2 release. This would be a great way to kickoff the blog.

### PR DESCRIPTION
**Pull request by abhi-g**
Posting the 0.2 release announcement on blog.

/fixes #80

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/130)

### CODE REVIEW

File: content/blog/announcing_kubeflow_0.2.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  - We also offer cloud-specific versions of these deployment scripts so that you can auto create a cluster if you don’t have one available. For example:
+    ```
+    export KUBEFLOW_VERSION=0.2.2
+    curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_VERSION}/scripts/gke/deploy.sh | bash
+    ```
+  - A central UI that gives you visibility into all the components running in your cluster - Central UI - To make it easier to navigate among components
+
+    ![Central UI Dashboard](../Central_UI_Screenshot.svg)
+
+  - A declarative deployment process that first creates Cloud resources including your Kubernetes cluster and then deploys Kubeflow. We have an example for GCP using Deployment Manager, and plan to add support for other clouds using tools like Terraform.
```
Comment (jlewi) [inline content/blog/announcing_kubeflow_0.2.md]: The bullet points don't render correctly.
https://screenshot.googleplex.com/f3LgLkdvdRb.png

This is an annoying thing about Netify markdown. Try removing the blank lines before/after the image link

Comment (abhi-g) [reply content/blog/announcing_kubeflow_0.2.md]: The bullet point rendering is an issue caused due to something in the CSS. We had noticed this earlier. Any bullet entry that happens to be longer than a single line ends up messing the placement of the bullet itself. We'll need to work with our designer to fix this. For example, see the first bullet point in the same list.

Comment (abhi-g) [reply content/blog/announcing_kubeflow_0.2.md]: For eg. if I remove that empty line before the image, here is how it looks: https://screenshot.googleplex.com/ixV42QBf8d3

**Pull request merged**